spec/integration/invalid_request_spec.rb

```diff
@@ -14,8 +14,8 @@
            "content-type" => "application/json",
          }
     expect(response.status).to eq(404)
-    expect(fake_logger.warnings.length).to eq(0)
-    expect(fake_logger.errors.length).to eq(0)
+    expect(fake_logger.warnings).to be_empty
+    expect(fake_logger.errors).to have_attributes(size: 1)
```

Comment:
Is there a behaviour change here in the framework, since we are changing the assertion from `length = 0` to `size = 1`?

Reply:
Yes, that’s because of this change: https://guides.rubyonrails.org/configuring.html#config-action-dispatch-debug-exception-log-level
Previously, the `ActionDispatch::DebugExceptions` middleware was logging exceptions using the `fatal` level, and now it’s using the `error` level.